handbook/HR/flexible-working-policy.md

```diff
@@ -0,0 +1,69 @@
+## Flexible Working Policy
```

Comment:
This doesn't seem relevant, or correct.  Perhaps link to here instead: https://github.com/DVELP/cookbook/tree/master/handbook#time-management